### PR TITLE
remove support for go1.3

### PR DESCRIPTION
**Pull request by thaJeztah**
### remove support for go1.3

go1.4 was released 8 Years ago, so it's very unlikely anyone is still using go1.3.

### rename files to regular name, and update buildtags

This allows us to remove the buildtag comment. Also updated the other buildtags to the current go format.

⚠️ I'm doing the rename in a separate commit, to assist Git history to continue keeping track.

### remove "os" import

os.Getppid() is an alias for syscall.Getppid(), which was already imported, so use it directly instead of the alias.

### CODE REVIEW

**Comment by thaJeztah** [comment]
@inconshreveable PTAL 🤗

**Review by inconshreveable — APPROVED** [review]
LGTM, thank you!

File: trap_windows.go [hunk 1]
```diff
@@ -1,98 +1,42 @@
...
-	dwFlags             uint32
-	szExeFile           [syscall.MAX_PATH]uint16
-}
-
-func getProcessEntry(pid int) (pe *processEntry32, err error) {
-	snapshot, _, e1 := CreateToolhelp32Snapshot.Call(th32cs_snapprocess, uintptr(0))
-	if snapshot == uintptr(syscall.InvalidHandle) {
-		err = fmt.Errorf("CreateToolhelp32Snapshot: %v", e1)
-		return
+func getProcessEntry(pid int) (*syscall.ProcessEntry32, error) {
```
Comment (thaJeztah) [inline trap_windows.go:8]: Thanks!

FWIW, I'm considering a pull request in golang.org/x/sys to see if the maintainers there are open to exporting the equivalent function there. I'll let you know if they do (and open a PR here)

**Comment by thaJeztah** [comment]
@inconshreveable thanks! Is there anything left to do on my hand, or is this OK to be merged?

If so; could you consider tagging a new release? After that I'll update some projects using it to use the new version 👍

**Comment by inconshreveable** [comment]
done, tagged v1.1

**Comment by thaJeztah** [comment]
@inconshreveable Thanks for merging! Erm.. would you be able to add a `v1.0.0` tag? As `v1.1` is not parsed as SemVer by `go mod`, so can't be used 😅

**Comment by inconshreveable** [comment]
pushed a v1.1.0 tag, lemme know if you need anything else!

**Comment by thaJeztah** [comment]
Looks like we're all set, thanks! I opened https://github.com/spf13/cobra/pull/1870

File: trap_windows.go (hunk 1 above)
Comment (thaJeztah) [reply trap_windows.go:8]: I opened https://github.com/golang/sys/pull/143  (https://go-review.googlesource.com/c/sys/+/459039) to see if the maintainers are open to exporting it 👍

**Pull request merged**